jupyter_console/tests/test_image_handler.py

```diff
@@ -11,7 +11,7 @@
 except ImportError:
     from mock import patch
 
-from IPython.kernel import KernelClient
+from ipykernel import KernelClient
```

Comment:
We don't expose `KernelClient` at the top level, so this needs to be `from ipykernel.client`